README.md

```diff
@@ -1,2 +1,57 @@
-# rhousehunter
-Simplify House-Hunting
+# R House Hunter 
+
+
+R package for searching for housing on Craigslist.
+
+-   Author: Ela Bandari, Junting He, Ling (Elina) Lin, Alex Truong
+
+
+## Overview
+
+Hunting for rentals can be an exhausting and frustrating experience in Canada, but this process can be made easy with a simple installation of our package. This Python package intends to facilitate the house hunting process by scraping the listing information from Craigslist and organizing the extracted data for the user. Instead of having to manually go on the website to catch up with individual new listings, the user will be updated through email with new results as per their selection criteria. 
+
+
+## Functions
+
+| Function Name | Input | Output | Description |
+|-----------|------------|---------------|------------------|
+| scraper | url | tibble | Scrape data from rental websites into a csv file.|
+| data_cleaner | tibble | tibble | Clean the extracted data |
```

Comment:
Hi Alex. I actually document the input and output as a data.frame.

Reply:
This is R so I think it is tibble :)